README.rst

```diff
@@ -560,6 +561,85 @@ Example:
         assert cucumbers["start"] - cucumbers["eat"] == left
 
 
+Using the datatable Fixture
```

Comment:
It's acutally an argument to the step execution, not a fixture.
If we call it fixture the users may expect it to be able to get it also via `request.getfixturevalue`